test/plots/polylinear.js

```diff
@@ -0,0 +1,45 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+const times = [
+  new Date(2013, 3, 5),
+  new Date(2013, 3, 11),
+  new Date(2013, 3, 14),
+  new Date(2013, 3, 16),
+  new Date(2013, 3, 18),
+  new Date(2013, 3, 21),
+  new Date(2013, 3, 27)
+];
+
+const events = [
+  { date: new Date(2013, 3, 5, 13, 0), text: "Initiate" },
+  { date: new Date(2013, 3, 11, 13, 0), text: "Begin" },
+  { date: new Date(2013, 3, 13, 20, 0), text: "Entry" },
+  { date: new Date(2013, 3, 15, 0, 0), text: "Test" },
+  { date: new Date(2013, 3, 16, 0, 0), text: "Drive" },
+  { date: new Date(2013, 3, 17, 8, 0), text: "Drive" },
+  { date: new Date(2013, 3, 18, 15, 0), text: "Brake" },
+  { date: new Date(2013, 3, 20, 10, 0), text: "Stop" },
+  { date: new Date(2013, 3, 23, 14, 0), text: "Shutdown" }
```

Comment:
These are local times; could we use UTC to avoid being dependent on the local timezone? (And then use d3.utcDays instead of d3.timeDays _etc._ below.)

I’d also prefer if you could change your default prettier settings to avoid the spaces inside the curly braces. 🙂